kong/plugins/aws-lambda/handler.lua

```diff
@@ -10,12 +10,13 @@ local cjson = require "cjson.safe"
 local public_utils = require "kong.tools.public"
 
 local string_find = string.find
+local string_fmt = string.format
 local ngx_req_get_headers = ngx.req.get_headers
 local ngx_req_read_body = ngx.req.read_body
 local ngx_req_get_uri_args = ngx.req.get_uri_args
 local ngx_req_get_body_data = ngx.req.get_body_data
-
-local CONTENT_TYPE = "content-type"
+local ngx_encode_base64 = ngx.encode_base64
+local var = ngx.var
```

Comment:
Do not localize `ngx.var` in this manner, if anything, localize it in function scope, but never in a module. Same thing with `ngx.ctx`:

https://github.com/openresty/lua-nginx-module#ngxctx

> Because of the metamethod magic, never "local" the ngx.ctx table outside your Lua function scope on the Lua module level due to worker-level data sharing.